### DISCUSSION TITLE
Suggestion | Translation text preview

### DISCUSSION THREAD

**Discussion by gpramvr** [zed-industries/zed#17558] [Feature Requests]
Hey team,

First of all, great IDE! Loving the experience so far.

I'm a beginner and use Ruby on Rails. In a project that I work on, we use i18n for handling all the localisation requirements. I've noticed that in RubyMine, there's a preview of the translation text in its default language.

| Expanded | Collapsed |
|--------|--------|
|![CleanShot 2024-09-08 at 21 13 46@2x](https://github.com/user-attachments/assets/3ac15ab4-8483-4326-9487-779336060386)|![CleanShot 2024-09-08 at 21 15 16@2x](https://github.com/user-attachments/assets/ecacd66e-9264-47bd-9c60-24ce679c5aeb)|

For people who work with a lot of UI like myself, this could be a pretty neat feature. I've noticed this feature only in RubyMine (perhaps its a JetBrains thing). I've heard there might be some external third party plugins for VSCode.

RubyMine also comes with a shortcut `cmd + option +shift + c`. It copies a key's path into the clipboard. Eg., `en.views.users.administrators.index.first_name`. That's pretty handy too.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
